libs/event_system/cringine/event_system/input/glfw/glfw_input_manager.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cringine/event_system/input/input_manager.hpp>
+
+namespace cringine::event_system::input
+{
+    class glfw_input_manager : public input_manager
+    {
+    public:
+        glfw_input_manager() = default;
+        virtual ~glfw_input_manager() = default;
```

Comment:
Clang-Tidy says: Prefer using 'override' or (rarely) 'final' instead of 'virtual'
So, change it to `~glfw_input_manager() override = default;`
Or just remove it